google_rest_api_generator/src/lib.rs

```diff
@@ -122,7 +121,7 @@ impl APIDesc {
         }
     }
 
-    fn generate(&self, auth_token: &str) -> TokenStream {
+    fn generate(&self) -> TokenStream {
```

Comment:
Can we change auth_token to scope? Along with corresponding changes to the examples and generated lib. With this change the scope requested is an empty list which makes the examples unusable for my testing.

Reply:
I have just pushed a change that makes it so that the scope is by default the same as in Byron/google-apis-rs (GET requests - read-only scope, otherwise the first scope as provided by discovery). Is it good enough or should I still add `scope` parameter?